File: project2-angular/src/app/models/plan.ts
class Plan {
    id: number;

    name: string;

    price: number;

    deviceLimit: number;

    constructor(id = 0, name = '', price = 0.0, deviceLimit=0){
        this.id = id;
        this.name = name;
        this.price = price;
        this.deviceLimit = deviceLimit; 
      }

}

export default Plan;

File: Project2/spring/project2/src/main/java/com/skillstorm/project2/plan/Plan.java
package com.skillstorm.project2.plan;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "plan")
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class Plan {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id;
    String name;
    float price;
    int deviceLimit;
}
